File: DSA/Sorting/bubblesort.py
def Bubble_sort(A):
    n = len(A)
    for i in range(n-1,0,-1):
        for j in range(i):
            if A[j] > A[j+1]:
                A[j],A[j+1] = A[j+1],A[j]
            # print(A)
            
A = [3,5,8,9,6,2]
print("Original list : ",A)
B = Bubble_sort(A)
print("***********************")
print("Sorted list : ",A)

File: DSA/DSA py and go/Sorting/bubblesort.go
package main

import "fmt"

func Bubble_sort(A []int) {
	n := len(A)
	for i := n - 1; i >= 0; i-- {
		for j := 0; j < i; j++ {
			if A[j] > A[j+1] {
				A[j], A[j+1] = A[j+1], A[j]
			}
		}
	}
}

func main() {
	A := []int{3, 5, 8, 9, 6, 2}
	fmt.Println("Original array : ", A)
	Bubble_sort(A)
	fmt.Println("Sorted array : ", A)
}
